scripts/packages/ui/src/components/notifications/ToastContainer.tsx

```
@@ -0,0 +1,50 @@
+ * ```
+ */
+export function ToastContainer() {
+  const dispatch = useAppDispatch();
+  const visibleNotifications = useAppSelector(selectVisibleNotifications);
+
+  const handleDismiss = (id: string) => {
+    dispatch(removeNotification(id));
+  };
+
+  return (
+    <div role="region" aria-label="Notifications">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)